homeworks/daryna.dereza_darynadereza/0-intro/README.md

```diff
@@ -0,0 +1,3 @@
+node v.18.18.1
+npm 9.8.1
+"C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Node.js\Node.js.lnk"
```

Comment:
Hi, @darynadereza, I assume, that this is path to the link to node from Start menu. We need here path to program itself. Try [where node](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/where) - analog of unix `which`